azure-iot-device/azure/iot/device/iothub/models/twin.py

```diff
@@ -0,0 +1,28 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""This module contains classes related to device twin and module twin functionality
+"""
+
+
+class Twin(object):
```

Comment:
>Twin(object): [](start = 6, length = 13)

Do we need metadata fields on this object? Or are desired/reported properties enough?

Reply:
We get some $version metadata (I think maybe only the top level), and that's it.  I think maybe there's value in adding it, but I'll have to chew on that.  

---
In reply to: [288771330](https://github.com/Azure/azure-iot-sdk-python-preview/pull/96#discussion_r288771330) [](ancestors = 288771330)

Reply:
Yeah, I'm not really sure either, I just know Twin can get complicated so I figured I'd raise it. We can always add it later if we need it.

---
In reply to: [288776802](https://github.com/Azure/azure-iot-sdk-python-preview/pull/96#discussion_r288776802) [](ancestors = 288776802,288771330)